var/spack/repos/builtin/packages/salmon/package.py

```diff
@@ -32,8 +32,10 @@ class Salmon(CMakePackage):
     homepage = "http://combine-lab.github.io/salmon/"
     url      = "https://github.com/COMBINE-lab/salmon/archive/v0.8.2.tar.gz"
 
+    version('0.9.1', '1277b8ed65d2c6982ed176a496a2a1e3')
     version('0.8.2', 'ee512697bc44b13661a16d4e14cf0a00')
 
+    depends_on('tbb')
```

Comment:
Is this needed for version 0.8.2 as well? Otherwise add `when=@0.9:`.

Reply:
That was my initial thought, but wasn't able to get 0.8.2 to properly install without it.